pkg/templates/templates_generated.go

```diff
@@ -5603,13 +5603,28 @@ function Start-InstallCalico {
     SetConfigParameters -RootDir $CalicoDir -OldString "CALICO_NETWORKING_BACKEND=`+"`"+`"vxlan`+"`"+`"" -NewString "CALICO_NETWORKING_BACKEND=`+"`"+`"none`+"`"+`""
     SetConfigParameters -RootDir $CalicoDir -OldString "KUBE_NETWORK = `+"`"+`"Calico.*`+"`"+`"" -NewString "KUBE_NETWORK = `+"`"+`"azure.*`+"`"+`""
 
-    GetCalicoKubeConfig -RootDir $CalicoDir -CalicoNamespace "kube-system" -SecretName "calico-windows"
+    $calicoNs = GetCalicoNamespace
+    GetCalicoKubeConfig -RootDir $CalicoDir -CalicoNamespace $calicoNs -SecretName "calico-windows"

```

Comment:
Just a reminder that once Calico version on Linux side  is updated to the latest, we would not need to apply a work around manifest to create `calico-windows` secret. The secret name should be updated to `calico-node` which is the default.

Reply:
Yes. I got your confirmation. Is anything needed to be updated here?

Reply:
No. Looking good!

Reply:
I know you have somewhere to track this future change, but could we add a comment here?

Reply:
In this guidance, we need step 4 as a workaround to test calico windows with old Linux calico version. We do not need it for using calico windows v3.17.1 with calico Linux v3.17.1. I think that we do not need such a comment for a step only used by an old test.